projects/RabbitMQ.Client/client/api/ReadonlyBasicProperties.cs

```diff
@@ -39,7 +39,7 @@ namespace RabbitMQ.Client
     /// <summary>
     /// AMQP specification content header properties for content class "basic"
     /// </summary>
-    public readonly struct ReadOnlyBasicProperties : IReadOnlyBasicProperties
+    public class ReadOnlyBasicProperties : IReadOnlyBasicProperties
```

Comment:
seal it?

Reply:
Thank you for noticing! @Tornhoof - always feel free to use the "Review" function in GitHub on a PR here. It should allow anyone to provide feedback.